StantUiIosLibrary/Classes/Cells/InternPersonCell.swift

```diff
@@ -33,24 +51,107 @@ public class InternPersonCell: UITableViewCell {
+        }
+        
         self.removeSubviews(self.titleLabel, self.descriptionLabel)
         
         let roundedAndFormattedImage = icon?.roundedReframedWith(size: iconSize)
-        
-        self.imageView?.image      = roundedAndFormattedImage
+        self.imageView?.image        = roundedAndFormattedImage
         
         positionElements()
         
         self.titleLabel.configure(text: title,
                                   size: titleSize,
                                   weight: .regular,
                                   color: .darkStant)
+        
         self.descriptionLabel.configure(text: description,
                                         size: descriptionSize,
                                         weight: .regular,
                                         color: .darkGrayStant)
     }
     
+    public func set(mailButton: UIButton? = nil,
+                    mailImage: UIImage,
+                    callButton: UIButton? = nil,
+                    callImage: UIImage) {
+        
+        self.mailImage.image = mailImage
+        self.callImage.image = callImage
+        
+        self.mailButton = mailButton
+        self.callButton = callButton
+        
+        positionOptionalElements()
+        styleButtonImages()
+    }
+    
+    private func styleButtonImages() {
+        if self.mailButton == nil {
+            self.mailImage.set(color: .darkGrayStant)
+        }
+        
+        if self.callButton == nil {
+            self.callImage.set(color: .darkGrayStant)
+        }
+    }
+    
+    private func positionOptionalElements() {
+        
+        self.addSubviews(mailImage, callImage)
+        
+        self.callImage.anchor(top: self.topAnchor,
+                              leading: nil,
+                              bottom: self.bottomAnchor,
+                              trailing: self.trailingAnchor,
+                              padding: UIEdgeInsets(top: 18,
+                                                    left: 0,
+                                                    bottom: 18,
+                                                    right: 16),
+                              size: CGSize(width: 20, height: 20))
+        self.mailImage.anchor(top: self.topAnchor,
+                              leading: nil,
+                              bottom: self.bottomAnchor,
+                              trailing: self.callImage.leadingAnchor,
+                              padding: UIEdgeInsets(top: 18,
+                                                    left: 0,
+                                                    bottom: 18,
+                                                    right: 16),
+                              size: CGSize(width: 20, height: 20))
+
+        if let callButton = self.callButton {
+            self.addSubview(callButton)
+            
+            self.callButton?.anchor(top: self.topAnchor,
+                                    leading: nil,
```

Comment:
Do you know that you dont need to send this nil here?

Reply:
You are right, just fixed it!